test/OpenTelemetry.Tests/Resources/ResourceTest.cs

```diff
@@ -158,7 +158,6 @@ public void CreateResource_NotSupportedAttributeTypes()
                 { "dynamic", new { } },
                 { "array", new int[1] },
                 { "complex", this },
-                { "float", 0.1f },
```

Comment:
since we are now no longer expecting an input of float to return an empty string, i have removed this test case.